### ISSUE TITLE
Display values

### ISSUE THREAD

**Issue by gdcumming** [gfmoore/ESCI-JS#9]
In Panel 4, display values of M, s. I suggest: ‘Latest sample  M [value]  s [value]’ all on one line.

Display in Panel 4 of the two values of MoE seems to me fine.

Panel 5, Mean Heap: checkboxes are fine. ESCI displays values for (1) ‘Mean heap SE’, (2) ‘Curve SE’, and (3) ‘Number [of means] in mean heap’. I like that, if practical. Then 1 and 2 refer to the means actually stacked in the mean heap, and 2 is calculated knowing N and sigma. It’s probably worth displaying also the number of means in the mean heap—then we can see that heap mean is first non-zero when that number is 1, and heap SE when it’s 2 (as ESCI does).

In Panel 7, we have ‘Heap mean [value]   Heap se [value]’  It’s good to have the heap mean (ESCI doesn’t), but these two values are better in Panel 5, because they relate to the heap, and because they are calculated from the means in the heap, not all the means in the current run, as are the other things in Panel 7. Use ‘SE’ not ‘se’.

Perhaps consider the layout of checkboxes, labels and values in Panels 4 and 5. Is it practical for these to be similar?

Panel 6. Make ‘C’ italic.

**Comment by gfmoore** [comment]
__In Panel 4, display values of M, s. I suggest: ‘Latest sample M [value] s [value]’ all on one line.__

Is this okay now? version 0.3.18

__ Panel 5, Mean Heap: checkboxes are fine. ESCI displays values for (1) ‘Mean heap SE’, (2) ‘Curve SE’, and (3) ‘Number [of means] in mean heap’. I like that, if practical. __ 
&
__Panel 7, we have ‘Heap mean [value] Heap se [value]’ It’s good to have the heap mean (ESCI doesn’t), but these two values are better in Panel 5, because they relate to the heap, and because they are calculated from the means in the heap, not all the means in the current run, as are the other things in Panel 7. Use ‘SE’ not ‘se’.__

This has confused me. Okay so I've moved "heap mean" and "Heap se"  to panel 5 and renamed them to "Mean heap M" and "Mean heap SE". Is that correct?

Added 'Curve SE' label, but not calculated it yet - because somewhere else I recall I have to do some work on the curve.

The "Number  in the mean heap" label added, ~but not calculated yet~ actually this was easy because I had already calculated the number in the heap. (Note this is only those in the heap and does not include those those dropping into the heap, which I think is correct.)

__Panel 6. Make ‘C’ italic.__

Done

__Is it practical for these to be similar?__

I'll send you a private message on this topic, but for now, though I'm happy to add, change things, I don't think I've got the time for that yet. :(
